Src/BootCamp.Chapter/MakeStuffWithInput.cs

```diff
@@ -0,0 +1,243 @@
+                            final += corner;
+                        }
+                        else if (IsThisHorizontal(iHeight, kWidth))
+                        {
+                            final += horizontal;
+                        }
+                        else if (IsThisVertical(iHeight, kWidth))
+                        {
+                            final += vertical;
+                        }
+                        else if (IsThisText(iHeight, kWidth))
+                        {
+                            final += WhatLetterToBePlaced(iHeight, kWidth);
+                        }
+                        else if (IsThisWhiteSpace(iHeight, kWidth))
+                        {
+                            final += " ";
+                        }
+                    }
+                    final += Environment.NewLine;
+                }
+                return final;
+            }
+            bool IsThisCorner(int iHeight, int kWidth)
+            {
+                // corners = 0 + 0 =====  0 + width === height + 0 === height + width
+                if ((iHeight == 0 && kWidth == 0) || (iHeight == 0 && kWidth == width)||
+                    (iHeight == height && kWidth == 0)|| (iHeight == height && kWidth == width))
+                {
+                    return true;
+                }
+
+                return false;
+            }
+            bool IsThisHorizontal(int iHeight, int kWidth)
+            {
+                // horizontal = height 0 !& width 0 !& width maxwidth || height max !& width 0 !& width maxwidth
+                if ((iHeight == 0 && !(kWidth == 0 || kWidth == width)) || (iHeight == height && !(kWidth == 0 || kWidth == width)))
+                {
+                    return true;
+                }
+                return false;
+            }
+            bool IsThisVertical(int iHeight, int kWidth)
+            {
+                // vertical oposite of horizontal
+                if ((kWidth == 0 && !(iHeight == 0 || iHeight == height)) || (kWidth == width && !(iHeight == 0 || iHeight == height)))
+                {
+                    return true;
+                }
+                return false;
+            }
+            bool IsThisWhiteSpace(int iHeight, int kWidth)
+            {
+                if (kWidth != 0 && kWidth != width && iHeight != 0 && iHeight != height)
+                {
+                    return true;
+                }
+                return false;
+            }
+            bool IsThisText(int iHeight, int kWidth)
+            {
+                int numberOfTextLines = splitmessage.Length;
+                int heightOfFirstTextLine = 1 + padding;
+                int textRow = iHeight - heightOfFirstTextLine;
+
+                if (iHeight >= heightOfFirstTextLine && iHeight < heightOfFirstTextLine + numberOfTextLines)
+                {
+                    if (kWidth >= heightOfFirstTextLine && kWidth < heightOfFirstTextLine + splitmessage[textRow].Length)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Merge this if statement with the enclosing one.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5123803)